src/AutoRest.CSharp/Common/Output/Models/Requests/RestClientMethod.cs

```diff
@@ -62,5 +62,10 @@ public List<Parameter> PathParameters
                 ).ToList();
             }
         }
+
+        public bool IsByIdMethod()
+        {
+            return this.Operation?.Requests.FirstOrDefault()?.Protocol.Http is HttpRequest httpRequest && this.Parameters.Count() > 0 && $"/{{{this.Parameters[0].Name}}}".Equals(httpRequest.Path);
+        }
```

Comment:
Can we remove this management sdk specific method from common to somewhere inside `AutoRest.CSharp/Mgmt/`?

Reply:
Good catch. Moved to `MethodExtensions.cs`.